### PR TITLE
add a namespace scoped example for e2e

### PR DESCRIPTION
**Pull request by chetan-rns**
**What type of PR is this?**

**What does this PR do / why we need it**:
Add a namespace scoped example to be consumed in the e2e tests

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**Test acceptance criteria**:

* [ ] Unit Test
* [ ] E2E Test

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: test/examples/nginx/nginx.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      app: nginx
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.16.1
```
Comment (amitkrout) [inline test/examples/nginx/nginx.yaml:17]: @chetan-rns I think OpenShift won't allow root image. So the containers image path would be ```nginxinc/nginx-unprivileged:1.16.1```

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: test/examples/nginx/nginx.yaml (hunk 1 above)
Comment (amitkrout) [reply test/examples/nginx/nginx.yaml:17]: Holding this pr till the review comment is not addressed 
/hold

Comment (chetan-rns) [reply test/examples/nginx/nginx.yaml]: Tried nginx:1.16.1 and it was deployed successfully. I've updated the image any way

**Pull request merged**